tests/validator/test_validation_graph.py

```diff
@@ -414,7 +419,15 @@ class DummyExecutionEngine:
             )
 
         graph = ValidationGraph(execution_engine=execution_engine)
-        graph.resolve_validation_graph(**call_args)
+        resolved_metrics: Dict[Tuple[str, str, str], MetricValue]
+        aborted_metrics_info: Dict[
+            Tuple[str, str, str],
```

Comment:
Could we make `Tuple[str, str, str]` a `namedtuple` instance? Can be an future effort/ticket just brainstorming.

Reply:
@cdkini Yes, and I think we can/should go even further; as mentioned above, this function will go through a couple of simplifications, due to its relevance for Batch-Metric Store.